docs/notes/2.26.x.md

```diff
@@ -20,6 +20,9 @@ New kubernetes backend! See [docs](https://www.pantsbuild.org/stable/docs/kubern
 
 ### Deprecations
 
+`[pex-cli].version` format `"v2.33.4"` is deprecated in favor of `"2.33.4"`.
```

Comment:
Ah, we often call out this sort of thing in the Python section specifically too.

So maybe this could be a really short bullet point here (e.g. "`[pex-cli].version` and `known_versions` should no longer include a `v` prefix") to make it easier to skim (if/when we have more), and then have this longer content in the Python section.